### PR TITLE
Enable deep zoom artifact creation from rgb bytes

### PR DESCRIPTION
**Pull request by ronaldcurtis**
Enables the creation of deep zoom artifacts from raw RGB8 slices.

### CODE REVIEW

File: src/bin/dzi.rs [hunk 1]
```diff
@@ -16,12 +16,10 @@ pub fn main() {
...
-        Ok(ic) => {
-            match ic.create_tiles() {
-                Ok(_) => {},
-                Err(e) => {
-                    eprintln!("Could not tile image:\n\t {:?}", e);
-                }
+        Ok(ic) => match ic.create_tiles() {
+            Ok(_) => {}
+            Err(e) => {
+                eprintln!("Could not tile image:\n\t {:?}", e);
```
Comment (ronaldcurtis) [inline src/bin/dzi.rs:22]: A lot of these diffs are just form `cargo fmt`

File: test_data/expected/test.dzi
Comment (ronaldcurtis) [inline test_data/expected/test.dzi:1]: Added the expected artifacts to version control so we can verify they are correctly generated in tests.

**Comment by jayvdb** [comment]
ping @n-k

**Comment by n-k** [comment]
Great patch, thanks! 
LGTM

I will try to release a new crate version over the weekend.

**Pull request merged**